site/content/user-guide/amqp-adapter.md

```diff
@@ -275,6 +275,35 @@ The table below provides an overview of the properties that must be set on a com
 | *correlation-id* | yes             | *properties*             | *string*     | MUST contain the value of the *correlation-id* property of the command request message. |
 | *status*         | yes             | *application-properties* | *integer*    | MUST contain a status code indicating the outcome of processing the command at the device (see [Command & Control API]({{< ref "/api/Command-And-Control-API.md" >}}) for details). |
 
+### Examples
+
+The AMQP Command-line client can be used to receive and send a command response message from/to a consumer application. The client is used to simulate a device to receive commands and for sending command responses back to the application. The command consumer client is used to simulate an application sending a command to the device and receiving a command response from the device.
+
+Start the AMQP Command-line client, as follows:
+
+    $ /hono/cli/target$ java -jar hono-cli-0.9-M1-SNAPSHOT-exec.jar --spring.profiles.active=amqp-adapter-cli --subscribe=true
+    $ 04:13:49.651 [vert.x-eventloop-thread-0] INFO  org.eclipse.hono.cli.AmqpSend - Device is now ready to receive commands
+
+After successfully starting the client, a message indicating that the device is ready to receive commands will be printed to standard output. The device is now waiting to receive commands from applications. To send a command to the device, open a new terminal shell and start the command consumer, as shown below:
+
+    $ /hono/cli/target$ mvn spring-boot:run -Drun.arguments=--hono.client.host=localhost,--hono.client.username=consumer@HONO,--hono.client.password=verysecret -Drun.profiles=command,ssl
+    >>>>>>>>> Enter name of command for device [DEFAULT_TENANT:4711] (prefix with 'ow:' to send one-way command):
+	setBrightness
+	>>>>>>>>> Enter command payload:
+	some-payload
+	>>>>>>>>> Enter content type:
+	text/plain
+
+Once the command consumer starts successfully, enter a command name, payload and content type of the command to send to the device.
+
+After sending the command, the device (i.e AMQP Command-line client) will printout the command message received from the consumer application and send back the response message. The command consumer prints the payload of the command response message to standard output, as shown below:
+
+    $ 04:15:40.210 [vert.x-eventloop-thread-0] INFO  org.eclipse.hono.cli.Commander - Received Command response : OK: setBrightness
+
+Finally, the device shows that the command response message has been received by the application by printing out the disposition to standard output:
+
+    $ Accepted{}
```

Comment:
the `$` character is a suffix to the current working directory path in the shell, i.e. if I am in directory `/home/hono` then the shell usually prints `/home/hono$` before the prompt.
If you assume that the user is working in directory `/hono/cli/target` (as you do above), then the shell will print
```
/hono/cli/target$ 
```
but not 
```
$ /hono/cli/target$
```
can you please fix this accordingly in the example?